src/components/DeleteModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+
+import Button from './Button';
+import { DeleteModalProps } from './types';
+
+import { useDeleteEmployeeMutation } from 'services/api';
+
+const DeleteModal: FC<DeleteModalProps> = (props) => {
+    const { setDelete, clickedId } = props;
+
+    const [deleteEmployee] = useDeleteEmployeeMutation();
+
+    const handleDeleteEmployee = (clickedEmployeeId: number) => {
+        deleteEmployee(clickedEmployeeId);
+        setDelete(false);
+    };
+
+    return (
+        <div className='fixed inset-0  h-full w-full overflow-y-auto  bg-gray-200/10'>
+            <div className='relative inset-x-36 top-60 mx-auto h-[209px] w-[400px] rounded-md border
+            bg-white p-5 text-center opacity-100 shadow-lg'>
+                <p className='p-1 text-2xl font-normal'>Are You Sure?</p>
+                <p className='p-1 text-lg font-normal'>Do you really want to delete employee?</p>
```

Comment:
Keep the text in a json file. Define and use it as a variable. It will help reuse the component. In case we want to delete a department we can fetch the corresponding text from the json and pass it along with props.